tests/test_providers/test_datetime.py

```diff
@@ -113,6 +114,13 @@ def test_datetime(self, _datetime, start, end, humanized, _type):
             year = int(dt.split(' ')[2])
             assert year == 2018
 
+        if humanized == True:
```

Comment:
Unneeded condition.

Reply:
If humanized is passed as `True` there will be no `tzinfo` attribute and the proceeding lines will result in an error. This `if ... pass` makes sure `tzinfo` isn't checked when the datetime is humanized. 

Maybe I am missing something, but I can't seem to find a better way to do this. Did you have anything in mind?